lib/traject/readers/folio_postgres_reader.rb

```diff
@@ -362,7 +362,8 @@ def contents_sql_query(conditions, addl_from: nil)
                                 'hrid', parentInstance.jsonb ->> 'hrid',
                                 'title', parentInstance.jsonb ->> 'title'
                               ),
-                              'holding', jsonb_build_object('effectiveLocationId', parentHolding.jsonb ->> 'effectiveLocationId'),
+                              'holding', jsonb_build_object('effectiveLocationId', parentHolding.jsonb ->> 'effectiveLocationId',
+                                                            'callNumber', parentHolding.jsonb ->> 'callNumber'),
```

Comment:
What part of https://github.com/sul-dlss/SearchWorks/issues/3200 needs the holdings call number? I think we actually need the item call number (plus the volume, enumeration, etc) to get the call number Darcy's showing in:

<img width="393" alt="Screenshot 2024-03-04 at 07 51 16" src="https://github.com/sul-dlss/searchworks_traject_indexer/assets/111218/63427d50-b84c-45da-9f6e-989350723e57">

At least, in the example Darcy provided, in the librarian view the holding call number for https://searchworks.stanford.edu/view/2307563 is just the base call  number:
<img width="335" alt="Screenshot 2024-03-04 at 07 52 12" src="https://github.com/sul-dlss/searchworks_traject_indexer/assets/111218/af3a5fa8-c003-4ded-b615-478f649904ea">

Reply:
@cbeer Thank you. I also wonder if this would be better if we added a `boundWith` into the `item_display_struct`.